spec/models_test/like_spec.rb

```diff
@@ -0,0 +1,10 @@
+require '../rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.new }
```

Comment:
- [ ] I encounter some issues when trying to run your tests. It has to do with the way you're requiring '../rails_helper'. See error below 👇 Please get rid of the '../' to fix this issue.
![image](https://user-images.githubusercontent.com/46839548/184511511-86c7aa18-4441-4b51-bc2c-e52191a8c4d1.png)